### PR TITLE
rework template

### PR DESCRIPTION
**Pull request by belopash**

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
   id: ID!
-  account: Account!
-  balance: BigInt!
-  timestamp: BigInt!
+  block: Int! @index
+  timestamp: DateTime! @index
+  extrinsic: String @index
+  from: Account!
+  to: Account!
+  amount: BigInt!
```
Comment (dzhelezov) [inline schema.graphql:15]: Add index here to filter big transfers

File: schema.graphql [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+  fee: BigInt
```
Comment (dzhelezov) [inline schema.graphql:16]: Should make the fee non-nullable and zero by default?

File: schema.graphql [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
 
-type HistoricalBalance @entity {
+type Transfer @entity {
   id: ID!
-  account: Account!
-  balance: BigInt!
-  timestamp: BigInt!
+  block: Int! @index
+  timestamp: DateTime! @index
+  extrinsic: String @index
```
Comment (dzhelezov) [inline schema.graphql:12]: txHash or extrinsicHash is a better name

**Review by dzhelezov — CHANGES_REQUESTED** [review]

File: schema.graphql (hunk 2 above)
Comment (belopash) [reply schema.graphql:16]: Not sure about that. We can't calculate fee for some old extrinsics. So null fee is like "fee is not defined".

File: schema.graphql (hunk 3 above)
Comment (belopash) [reply schema.graphql:12]: I think extrinsicHash will be better. txHash we use with evm.

Comment (dzhelezov) [reply schema.graphql]: Ok let's do `extrinsicHash`!

File: schema.graphql (hunk 2 above)
Comment (dzhelezov) [reply schema.graphql:16]: I think it complicates things for the API clients, so we can just set zero by default for this template (and add a field along the lines`The fee is calculated at the best effort and may be zero for some old extrinsics`).

**Review by dzhelezov — APPROVED** [review]
LGTM modulo the small changes in the comments

**Pull request merged**